applications/CMakeLists.txt

```diff
@@ -69,6 +69,8 @@ add_holohub_application(object_detection_torch HOLOSCAN_SAMPLE_APP)
 
 add_holohub_application(prohawk_video_replayer DEPENDS OPERATORS prohawk_video_processing)
 
+add_holohub_application(qt DEPENDS OPERATORS qt_video npp_filter)
```

Comment:
We need to have an explicit name for the application. "qt_video_replayer" ?

Reply:
Good idea, change the name.